Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -1081,7 +1081,7 @@ void VideoConfigDiag::PopulatePostProcessingShaders()
   if (shaders.empty())
     return;
 
-  choice_ppshader->AppendString(_("(off)"));
+  choice_ppshader->AppendString(_("Off"));
```

Comment:
Okay. I am going to leave this for now to see if anyone else has an opinion on it. Personally I do like this change as it looks neater especially when comparing to the stereoscopic 3D dropdown which is in the same tab.

Also if it does get reverted are you okay keeping it capitalized?